// PDB:  qcqdss7280.pdb
// PDB:  Last Updated :2022-05-17:11:06:06:456 (UTC) [tracepdb]
2fa33723-ebbb-347e-4708-655859ff0b58 remoteETM // SRC=remote_etm.c MJ= MN=
#typev remote_etm_c60 10 "%0QMI Client context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=Remote_etm_init_default_data
{
}
#typev remote_etm_c60 10 "%0QMI Client context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=Remote_etm_init_default_data$fin$0
{
}
#typev remote_etm_c148 14 "%0InValid Remote ETM state %10!d! hence bugcheck" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=checkForValidRemoteETMState
{
state, ItemLong -- 10
}
#typev remote_etm_c408 37 "%0qmi_client_release failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c402 36 "%0qmi_client_send_msg_sync response succeed with resp error 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
setStateResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c397 35 "%0qmi_client_send_msg_sync response failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
setStateResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c392 34 "%0qmi_client_send_msg_sync failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c388 33 "%0qmi_client_send_msg_sync failed with qmiErr 0x%10!x!, Service not available" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c363 32 "%0Failed to Wait on KeWaitForSingleObject" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
}
#typev remote_etm_c352 31 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
}
#typev remote_etm_c343 30 "%0pRemoteETMContext is NULL in enableRemoteETM" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
}
#typev remote_etm_c336 29 "%0workItemContext is NULL in enableRemoteETM" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM
{
}
#typev remote_etm_c408 37 "%0qmi_client_release failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c402 36 "%0qmi_client_send_msg_sync response succeed with resp error 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
setStateResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c397 35 "%0qmi_client_send_msg_sync response failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
setStateResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c392 34 "%0qmi_client_send_msg_sync failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c388 33 "%0qmi_client_send_msg_sync failed with qmiErr 0x%10!x!, Service not available" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c363 32 "%0Failed to Wait on KeWaitForSingleObject" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
}
#typev remote_etm_c352 31 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
}
#typev remote_etm_c343 30 "%0pRemoteETMContext is NULL in enableRemoteETM" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
}
#typev remote_etm_c336 29 "%0workItemContext is NULL in enableRemoteETM" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=disableRemoteETM$fin$0
{
}
#typev remote_etm_c308 27 "%0pQmiClient is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
}
#typev remote_etm_c311 28 "%0enableRemoteETM Exit" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
}
#typev remote_etm_c289 26 "%0qmi_client_send_msg_sync response succeed with resp error 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
setETMConfigResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c284 25 "%0qmi_client_send_msg_sync response failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
setETMConfigResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c279 24 "%0qmi_client_send_msg_sync failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c259 23 "%0qmi_client_send_msg_sync response succeed with resp error 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
setStateResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c254 22 "%0qmi_client_send_msg_sync response failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
setStateResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c249 21 "%0qmi_client_send_msg_sync failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c231 20 "%0remote_etm.c::%!FUNC! qmi_client_init_instance success err=%10!d!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c225 19 "%0remote_etm.c::%!FUNC! qmi_client_init_instance failed err=%10!d!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c201 18 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
}
#typev remote_etm_c190 17 "%0pRemoteETMContext is NULL in enableRemoteETM" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
}
#typev remote_etm_c183 16 "%0workItemContext is NULL in enableRemoteETM" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
}
#typev remote_etm_c179 15 "%0enableRemoteETM Entry" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM
{
}
#typev remote_etm_c308 27 "%0pQmiClient is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
}
#typev remote_etm_c311 28 "%0enableRemoteETM Exit" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
}
#typev remote_etm_c289 26 "%0qmi_client_send_msg_sync response succeed with resp error 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
setETMConfigResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c284 25 "%0qmi_client_send_msg_sync response failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
setETMConfigResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c279 24 "%0qmi_client_send_msg_sync failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c259 23 "%0qmi_client_send_msg_sync response succeed with resp error 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
setStateResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c254 22 "%0qmi_client_send_msg_sync response failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
setStateResp.resp.error, ItemLong -- 10
}
#typev remote_etm_c249 21 "%0qmi_client_send_msg_sync failed with qmiErr 0x%10!x!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c231 20 "%0remote_etm.c::%!FUNC! qmi_client_init_instance success err=%10!d!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c225 19 "%0remote_etm.c::%!FUNC! qmi_client_init_instance failed err=%10!d!" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
qmiErr, ItemLong -- 10
}
#typev remote_etm_c201 18 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
}
#typev remote_etm_c190 17 "%0pRemoteETMContext is NULL in enableRemoteETM" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
}
#typev remote_etm_c183 16 "%0workItemContext is NULL in enableRemoteETM" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
}
#typev remote_etm_c179 15 "%0enableRemoteETM Entry" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=enableRemoteETM$fin$0
{
}
#typev remote_etm_c535 45 "%0REMOTE ETM context or device handle is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_deinit
{
}
#typev remote_etm_c535 45 "%0REMOTE ETM context or device handle is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_deinit$fin$0
{
}
#typev remote_etm_c648 49 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_disable
{
}
#typev remote_etm_c640 48 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_disable
{
}
#typev remote_etm_c648 49 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_disable$fin$0
{
}
#typev remote_etm_c640 48 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_disable$fin$0
{
}
#typev remote_etm_c594 47 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_enable
{
}
#typev remote_etm_c586 46 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_enable
{
}
#typev remote_etm_c594 47 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_enable$fin$0
{
}
#typev remote_etm_c586 46 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_enable$fin$0
{
}
#typev remote_etm_c810 58 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_in_use
{
}
#typev remote_etm_c803 57 "%0TPDA block context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_in_use
{
}
#typev remote_etm_c810 58 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_in_use$fin$0
{
}
#typev remote_etm_c803 57 "%0TPDA block context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_in_use$fin$0
{
}
#typev remote_etm_c989 66 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_qmi_supported
{
}
#typev remote_etm_c982 65 "%0pRemoteETMContext is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_qmi_supported
{
}
#typev remote_etm_c989 66 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_qmi_supported$fin$0
{
}
#typev remote_etm_c982 65 "%0pRemoteETMContext is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_qmi_supported$fin$0
{
}
#typev remote_etm_c873 60 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_reg_flag_set
{
}
#typev remote_etm_c866 59 "%0TPDA block context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_reg_flag_set
{
}
#typev remote_etm_c873 60 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_reg_flag_set$fin$0
{
}
#typev remote_etm_c866 59 "%0TPDA block context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_is_reg_flag_set$fin$0
{
}
#typev remote_etm_c500 44 "%0remote_etm.c::%!FUNC! WdfWorkItemCreate failed in remote_etm_disable" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe
{
}
#typev remote_etm_c487 43 "%0remote_etm.c::%!FUNC! WdfWorkItemCreate failed in remote_etm_enable" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe
{
}
#typev remote_etm_c475 42 "%0remote_etm.c::%!FUNC! received QMI serviceObj=%10!p!" //   IFRLOG=CustomlogHandle LEVEL=TRACE_LEVEL_INFORMATION FLAGS=TRACE_DRIVER FUNC=remote_etm_probe
{
Arg, ItemPtr -- 10
}
#typev remote_etm_c471 41 "%0Failed to create QDSSC  service object" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe
{
}
#typev remote_etm_c463 40 "%0QMI Client init default data failed" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe
{
}
#typev remote_etm_c450 39 "%0Failed to create synchronization lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe
{
}
#typev remote_etm_c439 38 "%0REMOTE ETM context or device handle is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe
{
}
#typev remote_etm_c500 44 "%0remote_etm.c::%!FUNC! WdfWorkItemCreate failed in remote_etm_disable" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe$fin$0
{
}
#typev remote_etm_c487 43 "%0remote_etm.c::%!FUNC! WdfWorkItemCreate failed in remote_etm_enable" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe$fin$0
{
}
#typev remote_etm_c475 42 "%0remote_etm.c::%!FUNC! received QMI serviceObj=%10!p!" //   IFRLOG=CustomlogHandle LEVEL=TRACE_LEVEL_INFORMATION FLAGS=TRACE_DRIVER FUNC=remote_etm_probe$fin$0
{
Arg, ItemPtr -- 10
}
#typev remote_etm_c471 41 "%0Failed to create QDSSC  service object" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe$fin$0
{
}
#typev remote_etm_c463 40 "%0QMI Client init default data failed" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe$fin$0
{
}
#typev remote_etm_c450 39 "%0Failed to create synchronization lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe$fin$0
{
}
#typev remote_etm_c439 38 "%0REMOTE ETM context or device handle is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_probe$fin$0
{
}
#typev remote_etm_c1178 77 "%0remote_etm.c::%!FUNC! QCCI De init completed" //   IFRLOG=CustomlogHandle LEVEL=TRACE_LEVEL_INFORMATION FLAGS=TRACE_DRIVER FUNC=remote_etm_qmi_cci_deinit
{
}
#typev remote_etm_c1164 76 "%0remote_etm.c::%!FUNC! QCCI init completed" //   IFRLOG=CustomlogHandle LEVEL=TRACE_LEVEL_INFORMATION FLAGS=TRACE_DRIVER FUNC=remote_etm_qmi_cci_init
{
}
#typev remote_etm_c912 62 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_reg_set_flag
{
}
#typev remote_etm_c904 61 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_reg_set_flag
{
}
#typev remote_etm_c912 62 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_reg_set_flag$fin$0
{
}
#typev remote_etm_c904 61 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_reg_set_flag$fin$0
{
}
#typev remote_etm_c1137 75 "%0REMOTE ETM is already enabled" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_filter
{
}
#typev remote_etm_c1129 74 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_filter
{
}
#typev remote_etm_c1121 73 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_filter
{
}
#typev remote_etm_c1137 75 "%0REMOTE ETM is already enabled" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_filter$fin$0
{
}
#typev remote_etm_c1129 74 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_filter$fin$0
{
}
#typev remote_etm_c1121 73 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_filter$fin$0
{
}
#typev remote_etm_c1037 69 "%0REMOTE ETM is already enabled" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_mode
{
}
#typev remote_etm_c1029 68 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_mode
{
}
#typev remote_etm_c1021 67 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_mode
{
}
#typev remote_etm_c1037 69 "%0REMOTE ETM is already enabled" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_mode$fin$0
{
}
#typev remote_etm_c1029 68 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_mode$fin$0
{
}
#typev remote_etm_c1021 67 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_mode$fin$0
{
}
#typev remote_etm_c951 64 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_qmi_service
{
}
#typev remote_etm_c943 63 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_qmi_service
{
}
#typev remote_etm_c951 64 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_qmi_service$fin$0
{
}
#typev remote_etm_c943 63 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_qmi_service$fin$0
{
}
#typev remote_etm_c1087 72 "%0REMOTE ETM is already enabled" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_route
{
}
#typev remote_etm_c1079 71 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_route
{
}
#typev remote_etm_c1071 70 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_route
{
}
#typev remote_etm_c1087 72 "%0REMOTE ETM is already enabled" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_route$fin$0
{
}
#typev remote_etm_c1079 71 "%0Failed to acquire lock" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_route$fin$0
{
}
#typev remote_etm_c1071 70 "%0REMOTE ETM context is NULL" //   LEVEL=TRACE_LEVEL_ERROR FLAGS=TRACE_DRIVER FUNC=remote_etm_set_route$fin$0
{
}
